Tweak how the sse2 registers are displayed to make them
easier to understand

http://bugs.gentoo.org/92559

--- gdb/amd64-tdep.c
+++ gdb/amd64-tdep.c
@@ -103,22 +103,22 @@
   { "fop", &builtin_type_int32 },
 
   /* %xmm0 is register number 40.  */
-  { "xmm0", &builtin_type_v4sf },
-  { "xmm1", &builtin_type_v4sf },
-  { "xmm2", &builtin_type_v4sf },
-  { "xmm3", &builtin_type_v4sf },
-  { "xmm4", &builtin_type_v4sf },
-  { "xmm5", &builtin_type_v4sf },
-  { "xmm6", &builtin_type_v4sf },
-  { "xmm7", &builtin_type_v4sf },
-  { "xmm8", &builtin_type_v4sf },
-  { "xmm9", &builtin_type_v4sf },
-  { "xmm10", &builtin_type_v4sf },
-  { "xmm11", &builtin_type_v4sf },
-  { "xmm12", &builtin_type_v4sf },
-  { "xmm13", &builtin_type_v4sf },
-  { "xmm14", &builtin_type_v4sf },
-  { "xmm15", &builtin_type_v4sf },
+  { "xmm0", &builtin_type_vec128i },
+  { "xmm1", &builtin_type_vec128i },
+  { "xmm2", &builtin_type_vec128i },
+  { "xmm3", &builtin_type_vec128i },
+  { "xmm4", &builtin_type_vec128i },
+  { "xmm5", &builtin_type_vec128i },
+  { "xmm6", &builtin_type_vec128i },
+  { "xmm7", &builtin_type_vec128i },
+  { "xmm8", &builtin_type_vec128i },
+  { "xmm9", &builtin_type_vec128i },
+  { "xmm10", &builtin_type_vec128i },
+  { "xmm11", &builtin_type_vec128i },
+  { "xmm12", &builtin_type_vec128i },
+  { "xmm13", &builtin_type_vec128i },
+  { "xmm14", &builtin_type_vec128i },
+  { "xmm15", &builtin_type_vec128i },
   { "mxcsr", &builtin_type_int32 }
 };
 
